Samples/3b-custom-transformations/generated/pipeline.yaml

```diff
@@ -51,20 +51,18 @@ swagger-document/individual/azure-validator:
     - azure-validator-individual
   inputs:
     - swagger-document/individual/transform
-swagger-document/azure-openapi-validator:
-  pluginName: azure-openapi-validator
+swagger-document/azure-openapi-linter:
+  pluginName: azure-openapi-linter
   configScope:
     - azure-validator-composed
   inputs:
     - swagger-document/transform
-    - swagger-document/azure-validator
-swagger-document/individual/azure-openapi-validator:
-  pluginName: azure-openapi-validator
+swagger-document/individual/azure-openapi-linter:
+  pluginName: azure-openapi-linter
   configScope:
     - azure-validator-individual
   inputs:
     - swagger-document/individual/transform
-    - swagger-document/individual/azure-validator
```

Comment:
This is autogenerated and for some reason the line gets deleted. @olydis any idea why this might happen?